lib/yoti/protobuf/main.rb

```diff
@@ -45,8 +45,9 @@ def application_profile(receipt)
 
       def extra_data(receipt)
         return nil unless valid_receipt?(receipt)
+        return nil if receipt['extra_data_content'] == ''
 
-        decipher_profile(receipt['extra_data'], receipt['wrapped_receipt_key']) if receipt['extra_data']
+        decipher_extra_data(receipt['extra_data_content'], receipt['wrapped_receipt_key']) if receipt['extra_data_content']
```

Comment:
Do we need the `if receipt['extra_data_content']` if we're already returning nil if empty on the line above?